static/src/stylesheets/email/_article.scss

```diff
@@ -187,4 +187,82 @@ blockquote {
     padding: 0;
 }
 
+// Supporter Merchandising Module
+
+.subscribe {
+    border-collapse: collapse;
+    background-color: #ffffff;
+    &.sc {
+        width: 100%;
+        max-width: 580px;
```

Comment:
Does it make sense to use the [`$max-width`](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/email/_article.scss#L1) variable defined at the top of this file?